core/variant/callable.cpp

```diff
@@ -327,12 +329,26 @@ Callable::operator String() const {
 	}
 }
 
+Callable Callable::create(const Variant &p_variant, const StringName &p_method) {
+	if (unlikely(p_method == StringName())) {
+		ERR_FAIL_V_MSG(Callable(), "Method argument to Callable::create method must be a non-empty string");
+	}
+
+	switch (p_variant.get_type()) {
+		case Variant::NIL:
```

Comment:
```suggestion
		case Variant::NIL:
			return Callable(ObjectID(), p_method);
```
No need to go through the operator if we know what we have